aca_lts_eval.py

```diff
@@ -130,24 +132,46 @@ def max_temp(time, stars, manvr_error):
     id_hash = hashlib.md5(np.sort(stars['AGASC_ID'])).hexdigest()
     if id_hash not in T_CCD_CACHE:
         # Get tuple of (t_ccd, n_acq) for this star field and cache
-        T_CCD_CACHE[id_hash] = t_ccd_warm_limit(
+        P2_minus_5 = t_ccd_warm_limit(
             date=time,
             mags=stars['MAG_ACA'],
             colors=stars['COLOR1'],
             halfwidths=halfwidths,
-            min_n_acq=(2, 8e-3),
+            min_n_acq=(2, 1e-5),
             cold_t_ccd=COLD_T_CCD,
             warm_t_ccd=WARM_T_CCD,
             model='spline')
+        if P2_minus_5[0] < -10.2:
+            P2_minus_3 = t_ccd_warm_limit(
+                date=time,
+                mags=stars['MAG_ACA'],
+                colors=stars['COLOR1'],
+                halfwidths=halfwidths,
+                min_n_acq=(2, 1e-5),
+                cold_t_ccd=COLD_T_CCD,
+                warm_t_ccd=WARM_T_CCD,
+                model='spline')
+            if P2_minus_3[0] > -10.2:
+                T_CCD_CACHE[id_hash] = prob_n_acq(acq_success_prob(
+                        date=time,
```

Comment:
For the case where the P2=3 solved t_ccd_temperature would be warmer than -10.2, calculate the probabilities at -10.2 and save.